source/contribute.rst

```diff
@@ -198,6 +214,31 @@ this in case you want to read more about this:
 - `Commit message style for git <https://commit.style/>`_
 - `A note about git commit messages <https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html>`_
 
+Once you are done, you can push your branch to your fork:
+
+.. code-block:: console
+
+    $ git push --set-upstream fork your-sql-support # use -u for short
+
+Creating the pull request
+-------------------------
+
+New pull requests are created with the repository's default branch as
+base branch, and that might not be what you want. Make sure to pick the
+correct branch when creating the pull request. If you do not, it can
+still be changed afterwards.
```

Comment:
That'a a bit hidden – on GitHub, you have to click "Edit" next to the pull request title to change the title and base branch